css/user-search.styl

```diff
@@ -13,3 +13,6 @@
     border: none !important
     box-shadow: none !important
     background: none !important
+
+  .Select-menu
+    height: 174px
```

Comment:
Looks good, thoughts on switching to `max-height` so it can contract when underfilled?